functions/__fisher_index_update.fish

```diff
@@ -10,17 +10,17 @@ function __fisher_index_update -a timeout
     set -l url $fisher_index
     set -l index $fisher_cache/.index.tmp
 
-    if test -z "$url"
+    if command test -z "$url"
 
-        # Force the server to return the latest copy of the index using a fake query string.
+        # Force the server to return the lacommand test copy of the index using a fake query string.
 
         set url https://raw.githubusercontent.com/fisherman/fisher-index/master/index\?(date +%s)
     end
 
-    if not curl --max-time $timeout -sS "$url" > $index
-        rm -f $index
+    if not command curl --max-time $timeout -sS "$url" > $index
```

Comment:
Unfortunately we won't be able to `command curl` yet, maybe in the future, but currently, we need to use the default `curl` in order to run some tests that basically "mock" `curl`.